mitmproxy/net/websockets/utils.py

```diff
@@ -88,3 +89,26 @@ def get_client_key(headers):
 
 def get_server_accept(headers):
     return headers.get("sec-websocket-accept", None)
+
+
+def make_extension(extension):
```

Comment:
I think we can remove all of this, and use the `finalize` and `accept` functions of `wsproto.extensions.PerMessageDeflate` instead.
It takes the raw header value, so we don't have to parse it ourself.